mypaint-brush.c

```diff
@@ -22,6 +22,18 @@
 #include <math.h>
 #include <assert.h>
 
+#define GETTEXT_PACKAGE "libmypaint"
+
+#ifdef HAVE_GETTEXT
```

Comment:
Regarding GLib: since d0e315f10e7f4e2f30da8bc8bdf3c10fa4eb43e2 we support `--without-glib`, although building with introspection (the default) turns GLib back on again.

There may not be much of a need for this any more. Thoughts? @jonnor, was it only used to support the previous non-MSYS2 Appveyor build?